### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by piotrze**

### CODE REVIEW

File: marketplace_builder/views/pages/release-notes/2018-07-30.liquid [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+converter: markdown
+metadata:
+  title: 2018 07 30 — Release Notes
+  description: New tags, deprecate graphql endpoints
+slug: release-notes/2018-07-30
+searchable: true
+---
+{{ page.metadata.description }}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/release-notes/2018-07-30.liquid]: header, paragraph or smth?

File: marketplace_builder/views/pages/release-notes/index.liquid [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+converter: markdown
+metadata:
+  title: Release Notes
+  description: New features and changes in platform
+slug: release-notes
+searchable: true
+---
+{{ page.metadata.description }}
+
+* [2018-07-30: New tags, deprecate graphql queries](/release-notes/2018-07-30)
```
Comment (pavelloz) [inline marketplace_builder/views/pages/release-notes/index.liquid]: New liquid tags, deprecate `render_form` tag, deprecate `log` graphql mutation

File: marketplace_builder/views/pages/release-notes/2018-07-30.liquid [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+---
+### {{ page.metadata.description }}
+
+## Improvements
+
+* **include_form tag**: new tag allows you to use normal liquid syntax to pass variables
+
+  {% raw %}
+  ```
+  {% render_form 'edit_user', resource_id: @user.id %}
```
Comment (lemingos) [inline marketplace_builder/views/pages/release-notes/2018-07-30.liquid]: can we deprecate render_form ?

**Comment by lemingos** [comment]
There is a link in footer, can you link this page?

**Review by lemingos — CHANGES_REQUESTED** [review]
Please link this page in footer

**Review by lemingos — APPROVED** [review]

File: marketplace_builder/views/pages/release-notes/2018-07-30.liquid (hunk 3 above)
Comment (pavelloz) [reply marketplace_builder/views/pages/release-notes/2018-07-30.liquid]: yes - deprecation doesnt mean removal. yet. ;)

**Pull request merged**